src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -29,6 +29,7 @@
                             <pre>{{ value.message|linebreaksbr }}</pre>
                         {% endfor %}
                     </p>
+                    <a href="{% url 'plugins:sendmail:send' organizer=request.event.organizer.slug event=request.event.slug %}?from_log={{ log.id }}">{% trans "Send a new email based on this" %}</a>
```

Comment:
> I think it is more reasonable when we use from_log={{ log.id }} instead of from_log={{ log.object_id }}

Sure, my bad :see_no_evil: 

> Please review the code :)

I'll do as soon as possible!